bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/system/ITarget.java

```diff
@@ -28,7 +26,7 @@
  */
 public abstract class ITarget implements Cloneable, Serializable {
```

Comment:
I think right now we only have ```Target``` => ```SPTarget``` conversions in the AGS part. So for now we get away with that shortcut.